doc/api/configuration.md

```diff
@@ -196,6 +258,29 @@ To unset:
 %config SqlMagic.polars_dataframe_kwargs = {}
 ```
 
+
+## `short_errors`
+
+Default: `True`
+
+Don't display the full traceback
+on SQL Programming Error
+
+When False, full traceback
+message is displayed
+
+```{code-cell} ipython3
+%config SqlMagic.short_errors = False
+```
+
+When True, only a partial error
```

Comment:
not partial error, it raises custom exceptions along with original error from DB. but no stack trace